promptsource/templates/bing_coronavirus_query_set/templates.yaml

```
@@ -4,69 +4,77 @@ templates:
     answer_choices: null
     id: 43332782-9e92-4bb2-94bf-28759f3fe181
     jinja: "This search query talks about the coronavirus and was published on {{Date}}.\
-      \ In what country was it issued ? \n{{Query}}\n|||\n{{Country}}"
+      \ In what country was it issued? \n{{Query}}\n|||\n{{Country}}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - BLEU
```

```suggestion
      - Accuracy
```
bleu seems weird cause there is only a fixed set of countries

---

I was expecting it to be BLUE as, most probably, it is easier for the model to generate the country rather than predicting it.